examples/get-started/pure-js/widgets/app.js

```diff
@@ -27,9 +28,29 @@ const INITIAL_VIEW_STATE = {
   pitch: 30
 };
 
-new Deck({
+const UI_WIDGETS = [
+  new ZoomWidget({style: widgetTheme}),
+  new CompassWidget({style: widgetTheme}),
+  new FullscreenWidget({style: widgetTheme})
+];
+
+function updatePopup(object) {
+  const widgets = [...UI_WIDGETS];
+  if (object) {
+    const position = object.geometry.coordinates;
+    const text = `${object.properties.name} (${object.properties.abbrev})`;
+    const style = {width: 200, boxShadow: 'rgba(0, 0, 0, 0.5) 2px 2px 5px'};
+    widgets.push(new PopupWidget({position, text, style}));
```

Comment:
Should the `PopupWidget` just take a callback instead, so we don't need to mess with it like this?